setup.py

```diff
@@ -78,7 +78,7 @@ def determine_version():
     "Natural Language :: English",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
```

Comment:
It could be interesting to check if there are other things that can be updated now that the oldest supported version is 3.6.

Reply:
format strings everywhere :-)